test/fixtures/testcontainer-utils/src/main/java/org/elasticsearch/test/fixtures/testcontainers/TestContainersThreadFilter.java

```diff
@@ -17,6 +17,8 @@
 public class TestContainersThreadFilter implements ThreadFilter {
     @Override
     public boolean reject(Thread t) {
-        return t.getName().startsWith("testcontainers-") || t.getName().startsWith("ducttape");
+        return t.getName().startsWith("testcontainers-")
+            || t.getName().startsWith("ducttape")
+            || t.getName().startsWith("ForkJoinPool.commonPool-worker-1");
```

Comment:
Those are coming from test container internals using `Stream#parallel`. I currently discussing this issue with the testcontainer team in their community slack channel.